### PR TITLE
Installation process

### PR DESCRIPTION
**Pull request by yamihe19**
Installation Process information

### CODE REVIEW

File: docs/INSTALLATION PROCESS.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+## 1. Install Operating System:
+In this step, we will thoroughly evaluate our documentation within two distinct scenarios:
+
+### a. Virtual Machine Setup:
+
+For the first scenario, we will conduct our documentation testing on virtual machines. Specifically, we will be utilizing Google Cloud Machines, a powerful cloud-based platform that facilitates the creation of customized instances. This includes tailoring various parameters such as the choice of operating system, regional availability, GPU allocation, and more, based on your precise requirements.
+
+By employing virtual machines, we can emulate diverse computing environments and ensure that our documentation is comprehensive and adaptable to a range of configurations. This allows us to verify the effectiveness of our installation instructions across various settings and configurations.
+
+b.
```
Comment (SoloSynth1) [inline docs/INSTALLATION PROCESS.md]: It seems this line got cut off early. Is this intentional? @yamihe19

File: docs/INSTALLATION PROCESS.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+## Setting Up a Machine with GPU
+
+### 1.Install Operating System:
+Choose a suitable operating system. Ubuntu or other Linux distributions are preferred for deep learning due to their compatibility and control. We will focus on Ubuntu and Debian for this project.
+### 2. Install GPU Drivers:
+Download and install the latest GPU drivers compatible with your GPU model. Visit the GPU manufacturer's website (e.g., NVIDIA or AMD) to get the appropriate drivers.
+
+### 3. Create CONDA Environment:
+Utilize package managers like pip  conda to create a controlled environment. This ensures a clean setup for your deep learning project. For this project we are going to use Miniconda. 
+### 3. Install CUDA Toolkit:
```
Comment (SoloSynth1) [inline docs/INSTALLATION PROCESS.md]: `3.` is duplicated.

**Review by SoloSynth1 — COMMENTED** [review]
@yamihe19 Please see the comments. Thanks!

File: docs/INSTALLATION PROCESS.md (hunk 1 above)
Comment (yamihe19) [reply docs/INSTALLATION PROCESS.md]: It is intentional.

Comment (SoloSynth1) [reply docs/INSTALLATION PROCESS.md]: OK no problem! Resolving this issue.

File: docs/INSTALLATION PROCESS.md (hunk 2 above)
Comment (SoloSynth1) [reply docs/INSTALLATION PROCESS.md]: Resolved in 44586720b1eca28d9ce408df88a50d9189bb93c0, closing this comment.

**Comment by SoloSynth1** [comment]
All conversations have been resolved. LGTM. Merging to `main`.

**Review by SoloSynth1 — APPROVED** [review]

**Pull request merged**